OverlayPlugin.Core/MemoryProcessors/ClientFramework/ClientFrameworkMemory655ClientStructs.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Runtime.InteropServices;
+
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.ClientFramework
+{
+    partial class ClientFrameworkMemory655 : ClientFrameworkMemory, IClientFrameworkMemory655
+    {
+        // Due to lack of multi-version support in FFXIVClientStructs, we need to duplicate these structures here per-version
+        // We use FFXIVClientStructs versions of the structs because they have more required details than FFXIV_ACT_Plugin's struct definitions
```

Comment:
This line comment is irrelevant, FFXIV_ACT_Plugin doesn't have a struct for this.